lib/screens/utils/commonWidgets/boton_azul.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+class BotonAzul extends StatelessWidget {
+  final String text;
+  final VoidCallback onPressed;
+  final double sizeWidth;
+  const BotonAzul(
+      {required this.text,
+      required this.onPressed,
+      this.sizeWidth = double.infinity});
+
+  @override
+  Widget build(BuildContext context) {
+    return RaisedButton(
```

Comment:
This widget is deprecated, use ElevatedButton instead:
- https://docs.google.com/document/d/1yohSuYrvyya5V1hB6j9pJskavCdVq9sVeTqSoEPsWH0/edit